src/components/SummitCard.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+
+import calendarIcon from '../../static/img/summit-landing/icons/calendar-icon.svg'
+import locationIcon from '../../static/img/summit-landing/icons/location-icon.svg'
+
+import '../style/modules/_summit-card.scss'
+
+const SummitCard = ({ background, date, location, notification = false }) => {
+    return (
+        <section className='summit-card-wrapper'>            
+            <div className="summit-card-image" style={{ backgroundImage: `url(${background}` }}></div>
+            <div className={`summit-card-info ${notification ? '' : 'no-notification'}`}>
+                <div className='summit-card-info-wrapper'>
+                    <span>
+                        <img src={calendarIcon} /> {date}
+                    </span>
+                    <span>
+                        <img src={locationIcon} /> {location} 
+                    </span>
+                </div>
+            </div>
+            {notification &&
+                <div className="summit-card-notification">
+                    <span>
+                        <b>Get ready!</b> The Call for presentations will open up in Q1 of 2024.
```

Comment:
this need to be passed on the object not hardcoded also we need a prop on the object for the get notified HREF

Reply:
instead of have a boolean (notification) u could said to pass a a single object , summit 
that has
summit = {
date,
location
notification: {
text,
link
}
}
being summit.notification object optional